src/main/resources/templates/lsp/stylelint-lsp/template.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Added the main language server template registered for the CSS file type and extensions.